MyCryptoHQ/MyCrypto
# WIP: Add Cookie/Analytics Notification

Solution to https://mycrypto.atlassian.net/browse/CORE-76.

Implements a display at the bottom of the screen that provides a way for users to acknowledge our cookie/analytics services.

# Testing

1. Load the home page.
2. The analytics notification should appear with `Cookies` already checked, and `Analytics` not checked.
3. Click "Accept".
4. The notification should disappear.
5. Refresh the page.
6. The notification should be absent.
7. Open Chrome DevTools -> click on the `Application` tab -> expand `Local Storage` on the left -> click the entry underneath.
8. Locate the entry named `MyCrypto Analytics`:
<img width="857" alt="screen shot 2018-11-20 at 6 53 37 pm" src="https://user-images.githubusercontent.com/10479826/48811943-a12ca380-ecf5-11e8-99a2-0b7cc35abae2.png">

9. The value for the entry should read `{ basic: true, analytics: false }`.

10. Click that entry and press `delete`.

11. The analytics widget should appear on the screen again.

12. Click the checkbox next to `Analytics`.

13. Press "Accept".

14. Follow steps 7. and 8.

15. This time, the value for the entry should read `{ basic: true, analytics: true }`.



# Screenshots

## Mobile view:
<img width="246" alt="screen shot 2018-11-20 at 6 33 21 pm" src="https://user-images.githubusercontent.com/10479826/48811474-9113c480-ecf3-11e8-821f-738820447570.png">

## Tablet view:
<img width="337" alt="screen shot 2018-11-20 at 6 33 34 pm" src="https://user-images.githubusercontent.com/10479826/48811475-9113c480-ecf3-11e8-9cc0-a5321f59bbfe.png">

## Desktop view:
<img width="1440" alt="screen shot 2018-11-20 at 6 34 58 pm" src="https://user-images.githubusercontent.com/10479826/48811477-9113c480-ecf3-11e8-97c8-f8e190916ac3.png">
